code/html/index.html

```diff
@@ -1,16 +1,34 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" dir="ltr">
 
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width , initial-scale=1">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```

Comment:
We don't want dependent on another server. Download the library and use it from your folder.

Reply:
Ok. Some small polishes ... if file is minified should be name xxx.min.js

Reply:
polished... minified files contain min word in the name.